[ENH] Add Twitter badge to README and link to website to landing page (#268)
bids-standard/bids-specification: src/index.md

To get started, [check out the introduction](01-introduction.md). If you'd like
more information on how to adapt your own datasets to match the BIDS
specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).

> For an overview of the BIDS ecosystem, visit the [BIDS homepage](bids.neuroimaging.io).

Author: ```suggestion
For an overview of the BIDS ecosystem, visit the [BIDS homepage](https://bids.neuroimaging.io).
```

> 

Author: ```suggestion
```

To get started, [check out the introduction](01-introduction.md). If you'd like
more information on how to adapt your own datasets to match the BIDS
specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).

> If you are interested in getting occasional updates on BIDS 

Reviewer: Sentence fragment. Guessing it was supposed to be removed.
```suggestion
```

To get started, [check out the introduction](01-introduction.md). If you'd like
more information on how to adapt your own datasets to match the BIDS
specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).

If you are interested in getting occasional updates on BIDS 

> please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit)

Reviewer: ```suggestion
please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit).
```

Does the mailing list not have a more standard subscription management? e.g., most listservs have a page where you can sign up and then go back and unsubscribe if you want. Using a Google Form makes it feel like it's going to just be a list in somebody's contacts, which is hard to get off of.

Author: at this time no. having a more standard subscription management can be good at this point before using it (e.g. newsletters)

May you have recommendations on one?

Author: ```suggestion

```

> You can also find us on Twitter! ![](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)

Reviewer: A badge here doesn't seem to add anything.

```suggestion
You can also find us on [Twitter](https://twitter.com/BIDSstandard]!
```

Author: I think a badge looks better than having a hyperlink to twitter

Author: applying your previous suggestion to get the badge directly linked to twitter

```suggestion
You can also find us on Twitter! [![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard)
```

Author: ```suggestion

```
